### ISSUE TITLE
"Add job" link throws 404 in current master branch

### ISSUE THREAD

**Issue by tofarley** [fsiddi/flamenco#134]
I could be missing something here, but using the current master branch, when I click the "Add Job" button in the webui I am greeted with a blank modal dialog that says "Modal title" at the top, has two separator bars (no template rendering between them as it should). With Cancel and Add Job buttons at the bottom.

On the dashboard console it says:
`[16/May/2015 22:57:12] "GET /jobs/add?_=1431816964526 HTTP/1.1" 404 -`

This occurs when I run the 4 services "out of the box" with no configuration updates. When I enable debugging in all of the config files, I get the error:

`DataTables warning: table id=jobs - Alax error. For more information about this error, please see http://datatables.net/tn/7`

I am using the default sqlite3 configuration. The databases have been created successfully (though the table name is 'job' not 'jobs' I believe jobs is just the model reference...)

I'm still investigating what this means, and will provide more information if I can.

**Comment by fsiddi** [comment]
Hello,
thanks for taking the time to write this report. The problem is simple: currently it's not possible to submit jobs from the web interface anymore. The only way is via the Flamenco blender add-on.

Of course we plan to bring back the functionality, but we did not have time yet.

I just committed the removal of the "Add" in 6d7b276d29a1bc9706150f07c7465c4d2351c30f.

I apologize for the time you spent trying to figure this out.
